cmd/dlt/cmd.go

```diff
@@ -23,15 +23,16 @@ import (
 	"github.com/openshift/moactl/cmd/dlt/idp"
 	"github.com/openshift/moactl/cmd/dlt/ingress"
 	"github.com/openshift/moactl/cmd/dlt/user"
-
+	v "github.com/openshift/moactl/cmd/validations"
 	"github.com/openshift/moactl/pkg/confirm"
 )
 
 var Cmd = &cobra.Command{
-	Use:     "delete RESOURCE [flags]",
-	Aliases: []string{"remove"},
-	Short:   "Delete a specific resource",
-	Long:    "Delete a specific resource",
+	Use:              "delete RESOURCE [flags]",
+	Aliases:          []string{"remove"},
+	Short:            "Delete a specific resource",
+	Long:             "Delete a specific resource",
+	PersistentPreRun: v.Validations,
```

Comment:
Same for `delete` as in `edit`, if the cluster exists, then the CF stack was already created and still exists

Reply:
Addressed and resolved